cms/plugins/appentities/director/models/Director.php

```diff
@@ -0,0 +1,56 @@
+<?php namespace Appentities\Director\Models;
+
+use October\Rain\Database\Model;
+use Appentities\Company\Models\Company;
+
+class Director extends Model
+{
+    use \October\Rain\Database\Traits\Validation;
+
+    public $table = 'apidata_directors';
+
+    public $rules = [];
+
+    public $fillable = [
+        'name',
+        'street',
+        'city',
+        'number',
+        'zip',
+    ];
+
+    public $belongsToMany = [
+        'companies' => [
+            Company::class,
+            'pivotModel' => CompanyDirector::class,
+            'table' => 'apidata_company_director_pivot',
+        ],
+    ];
+
+    /**
+     * Check if director exists
+     *
+     * @param array|Director $director
+     * @param boolean $returnId Return director id if exists
+     * @return boolean|int
+     */
+    public static function exists(Director|array $director, bool $returnId): bool|int
```

Comment:
toto co je, vsak sa da pouzit $query->exists

Reply:
pretoze k tejtu funkcii sa dostanem v mieste kde ja mam uz nejaky associativny array s tymi values toho directora a takto to je trocha krajsie